### DISCUSSION TITLE
Is the square pad the positive pad for the backlight LEDs?

### DISCUSSION THREAD

**Discussion by mjparme** [ZeroZeroOne-dev/00Key#11] [Q&A] [answered]
I can't get the backlight to come on and I just want to confirm the square pad was the positive pad for the LEDs. I suppose I should have asked before soldering them but square pad being positive is the general convention so just went with it.

**Comment by mjparme** [comment]
Ok, I switched one of the LED's around and the _round_ pad is positive. Bro...you're killing me :-) 

My bad for not asking first or taking a look at it in KiCAD.

**Comment by mjparme** [comment, marked as answer]
Sigh...I should have looked at the PCB in KiCAD first before soldering them in:

<img width="415" alt="image" src="https://user-images.githubusercontent.com/1580996/149248813-329db1ed-f32a-4cd4-9973-bad8f11c085b.png">

**Comment by ZeroZeroOne-dev** [comment]
Square pad usually is the cathode (-) so the short leg goes there 🙂

![image](https://user-images.githubusercontent.com/254635/149349605-aca9bbf3-9528-4fc8-ac04-7221afe141b4.png)

**Reply by mjparme** [reply]
That is an interesting graphic because I learned it just the opposite i.e. positive is generally square. On the Iris and Melody96 I soldered the square pad was positive. Although there is also no hard and fast rule and PCB designers can choose for themselves. So I should have looked at it in KiCad before assuming. Some markings on the PCB itself would be helpful as well 😉

I fixed it all up, I have one of these desoldering irons (https://www.amazon.com/gp/product/B094ZCRXMN) and it makes short work of mistakes like these. 

This keyboard is for my son (10 yrs old) and he put the LEDs in and soldered them by himself (after I told him to put the long leg in the square hole). He did a great job on the soldering and he was a little annoyed when we figured out I told him to put them in backwards. Doh!

**Reply by ZeroZeroOne-dev** [reply]
I think I'll write some build instructions soon so its more clear which orientation to put the LEDs in the PCB.

Glad you could fix it, hope your son and you are enjoying the build 😄
